neqo-crypto/build.rs

```diff
@@ -285,13 +284,69 @@ fn main() {
         };
         dynamic_link(libs);
     }
+    includes
+}
+
+fn setup_for_gecko() -> Vec<PathBuf> {
+    let mut includes: Vec<PathBuf> = Vec::new();
+    let libs = if env::consts::OS == "windows" {
+        &["nssutil3.dll", "nss3.dll", "ssl3.dll"]
+    } else {
+        &["nssutil3", "nss3", "ssl3"]
+    };
+    dynamic_link(libs);
```

Comment:
This block duplicates the block from the end of the setup_for_non_gecko().  Can you factor that out please?